packages/envied/lib/src/envied_base.dart

```diff
@@ -160,6 +165,10 @@ final class EnviedField {
   /// Escapes single quotes and newlines in the value.
   final bool? rawString;
 
+  /// A seed can be provided if the obfuscation randomness needs to remain
+  /// reproducible across builds.
```

Comment:
```suggestion
  /// A seed can be provided if the obfuscation randomness needs to remain
  /// reproducible across builds.
  /// **Note**: This will make the `Random` instance non-secure!
```